KodiAPI/Error.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Error.swift
+//  KodiAPI
+//
+//  Created by Pavel Prokofyev on 2/12/18.
+//  Copyright © 2018 Pavel Prokofyev. All rights reserved.
+//
+
+import Foundation
+
+enum KodiError : Error {
```

Comment:
stick to a single code style:
```suggestion
enum KodiError: Error {
```